Frameworks/CoreText/DWriteWrapper.mm

```diff
@@ -493,6 +507,29 @@ static void __GetGlyphRunDetails(_CTTypesetter* ts, CFRange range, CGRect frameS
                 line->_width += glyphRunDetails._dwriteGlyphRun[i].glyphAdvances[index];
             }
 
+            DWRITE_GLYPH_METRICS glyphMetrics[run->_dwriteGlyphRun.glyphCount];
+            THROW_IF_FAILED(run->_dwriteGlyphRun.fontFace->GetDesignGlyphMetrics(run->_dwriteGlyphRun.glyphIndices,
+                                                                                 run->_dwriteGlyphRun.glyphCount,
+                                                                                 glyphMetrics,
+                                                                                 run->_dwriteGlyphRun.isSideways));
+
+            CTFontRef font = static_cast<CTFontRef>([run->_attributes objectForKey:static_cast<NSString*>(kCTFontAttributeName)]);
+            CGFloat pointSize = kCTFontSystemFontSize;
+            if (font) {
+                pointSize = CTFontGetSize(font);
+            }
+
+            DWRITE_FONT_METRICS fontMetrics;
+            run->_dwriteGlyphRun.fontFace->GetMetrics(&fontMetrics);
+            CGFloat pointSizeUnitsPerEm = pointSize / fontMetrics.designUnitsPerEm;
```

Comment:
...?

pointSize / designUnitsPerEm
= (pt / em) / (design unit/em)
= pt / design unit

this float is not what you labeled it...